### DISCUSSION TITLE
Pull Request: Added ability to suppress suffix

### DISCUSSION THREAD

**Discussion by scodari** [open-webui/open-webui#3766] [General]
Hi, I've made a small change to add a WEBUI_NAME_SUFFIX  environmental variable, to allow the removal of the "(Open WebUI)" suffix after WEBUI_NAME.
The default behavior would keep the suffix.

Here you can see the commit: https://github.com/open-webui/open-webui/commit/3665613924e32dcbbd8042846b08ccc611e1f4a5

Can I create a pull request?

**Comment by justinh-rahb** [comment]
For that specifically, nope. it's done as it's done quite deliberately.

**Comment by jwics7** [comment]
They're greedy about it but there's ways around it.
